### PR TITLE
Feat(medusa-test-utils, authentication, customer, cart, payment, pricing, product, promotion): Add initModule util

### PR DESCRIPTION
**Pull request by pKorsholm**
**What**
- Add initModule util to load modules in integration tests using medusaApp

### CODE REVIEW

**Review by adrien2p — COMMENTED** [review]
Looks very nice, I would just return everything from the medusa app in case you need to access the query or run migration etc. 

cc @carlos-r-l-rodrigues

**Review by adrien2p — APPROVED** [review]
LGTM 🔥 just need to CI to pass

File: packages/customer/integration-tests/utils/get-init-module-config.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { Modules, ModulesDefinition } from "@medusajs/modules-sdk"
+
+import { DB_URL } from "./database"
+
+export function getInitModuleConfig() {
+  const authenticationModuleOptions = {
```
Comment (carlos-r-l-rodrigues) [inline packages/customer/integration-tests/utils/get-init-module-config.ts]:
```suggestion
  const moduleOptions = {
```

File: packages/authentication/integration-tests/utils/get-init-module-config.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { Modules, ModulesDefinition } from "@medusajs/modules-sdk"
+
+import { DB_URL } from "./database"
+
+export function getInitModuleConfig() {
+  const authenticationModuleOptions = {
```
Comment (carlos-r-l-rodrigues) [inline packages/authentication/integration-tests/utils/get-init-module-config.ts]:
```suggestion
  const moduleOptions = {
```
to facilitate the copy/paste 😆

**Review by carlos-r-l-rodrigues — APPROVED** [review]
Amazing. Thank you

File: packages/authentication/integration-tests/utils/get-init-module-config.ts (hunk 2 above)
Comment (pKorsholm) [reply packages/authentication/integration-tests/utils/get-init-module-config.ts]: you got me 😂

**Pull request merged**